api/types/plugins/logdriver/entry.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
```

Comment:
Stuff in `pkg/` tends to be a bit more open to change. The `LogEntry` type in particular should not be changed, or rather changed with extreme caution, which is why I put it in `api/types`... but honestly wherever people want to put it is fine.